codec-http2/src/main/java/io/netty/handler/codec/http2/Http2FrameCodec.java

```diff
@@ -51,6 +51,8 @@
     private ChannelHandlerContext ctx;
     private ChannelHandlerContext http2HandlerCtx;
 
+    // TODO(buchgr): If MAX_CONCURRENT_STREAMS is reached, send a Http2StreamClosedEvent for any streams in idle state
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1135)